pkg/operator/starter.go

```diff
@@ -210,8 +214,34 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 					"v3.11.0/openshift-apiserver/svc.yaml",
 					"v3.11.0/openshift-apiserver/sa.yaml",
 					"v3.11.0/openshift-apiserver/trusted_ca_cm.yaml",
+				},
+			},
+			{
+				Files: []string{
 					"v3.11.0/openshift-apiserver/pdb.yaml",
 				},
+				ShouldCreateFn: func() bool {
+					if !infraInformer.HasSynced() {
```

Comment:
I'd replace with `configInformers.Config().V1().Infrastructures().Informer().HasSynced()`. No need to intro a new variable.